### ISSUE TITLE
Include braille tables for romanian language in NVDA's braille settings

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#7847]
### Steps to reproduce:
1. Connect a braille display with NVDA and write following:
Romanian: Haș
2. Try to read the words on the Braille Display
3. Go to the last letter of the romanian word and try to read the single character with eSpeak

### Expected behavior:
The letter ș as well as other language specıfıc signs should automatically be properly dısplayed on a braılle dısplay. ș is for example dots 2, 4, 6 (grade 1).

### Actual behavior:
On the Braille Display there is a code generated (for romanian letter for example 'x219') instead of displaying the letter itself.

### System configuration:
NVDA version:
2017.4

NVDA Installed or portable:
Both

Other information:

Windows version:
Windows 10 Build 16299.64

Name and version of other software in use when reproducing the issue:
All

### Other questions:

Does the issue still occur after restarting your PC?
yes

Have you tried any other versions of NVDA?
Yes, 2017.3, 2017.1

**Comment by josephsl** [comment]
Hi, let us discuss this on Espeak NG and Libluis mailing lists please. Thanks.

**Comment by Adriani90** [comment]
Hey Joseph,

 

looking at the source code, the Romanian table seems to be there. I can find it under dist / louis / tables, as well as in “include / Liblouis” and “source / Louis”. However, in NVDA Braille settings I cannot choose Romanian table.

 

Do you have any advice how to solve this?

 

It could be the case for other languages as well.

 

 

Best

Adriani

 

 

 

Von: Joseph Lee [mailto:notifications@github.com] 
Gesendet: Montag, 11. Dezember 2017 17:59
An: nvaccess/nvda <nvda@noreply.github.com>
Cc: Adriani90 <adriani.botez@googlemail.com>; Author <author@noreply.github.com>
Betreff: Re: [nvaccess/nvda] Add braille tables for romanian language (#7847)

 

Hi, let us discuss this on Espeak NG and Libluis mailing lists please. Thanks.

**Comment by josephsl** [comment]
Hi, if the Romanian community says the table in Liblouis is ready for the public, that is when I or someone else will add it to the list. CC @LeonardDer

**Comment by josephsl** [comment]
CC @LeonardDer, can we do it as part of Liblouis 3.4.1 PR please? Thanks.

**Comment by leonardder** [comment]
Sure, I'll deal with that.

**Comment by leonardder** [comment]
@Adriani90: For the speech part of this issue, please consult the Espeak-ng mailing list or file an issue against their repo. I will only deal with braille as part of this issue, as the issue title also says so.

**Comment by Adriani90** [comment]
@leonardder, I have edited the issue accordingly. Thank you very much.

**Comment by Adriani90** [comment]
I have found a work around for #7884. Maybe you could have a look at that as well.. Thanks.

**Comment by Adriani90** [comment]
#7848
